package.json

```diff
@@ -38,6 +38,7 @@
     "mocha": "^9.1.3"
   },
   "dependencies": {
+    "commander": "^8.3.0",
```

Comment:
This is one of the most famous node package. We should explore it even more to add commands/subcommands.